### PR TITLE
Manually dispatch pipeline ElasticJob events when ShardingSphere-Proxy start

### PR DESCRIPTION
**Pull request by azexcy**
Changes proposed in this pull request:
  - Manually dispatch pipeline ElasticJob events if job `disable=false`

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java [hunk 1]
```diff
@@ -51,6 +61,22 @@ public void onInitialized(final String databaseName, final ContextManager contex
         PipelineContextManager.putContext(contextKey, new PipelineContext(modeConfig, contextManager));
         PipelineMetaDataNodeWatcher.getInstance(contextKey);
         ElasticJobServiceLoader.registerTypedService(ElasticJobListener.class);
+        dispatchEnablePipelineJobStartEvent(contextKey);
+    }
+    
+    private void dispatchEnablePipelineJobStartEvent(final PipelineContextKey contextKey) {
```
Comment (sandynz) [inline kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java:67]: Is there possible exception thrown in this method?

File: kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java [hunk 2]
```diff
@@ -51,6 +61,22 @@ public void onInitialized(final String databaseName, final ContextManager contex
...
+        JobConfigurationAPI jobConfigAPI = PipelineAPIFactory.getJobConfigurationAPI(contextKey);
+        List<JobBriefInfo> allJobsBriefInfo = PipelineAPIFactory.getJobStatisticsAPI(contextKey).getAllJobsBriefInfo()
+                .stream().filter(each -> !each.getJobName().startsWith("_")).collect(Collectors.toList());
+        for (JobBriefInfo each : allJobsBriefInfo) {
+            PipelineJobType pipelineJobType = PipelineJobIdUtils.parseJobType(each.getJobName());
```
Comment (sandynz) [inline kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java:72]: Variable name `pipelineJobType` and `jobConfiguration` could be improved

**Review by sandynz — CHANGES_REQUESTED** [review]

File: kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java (hunk 1 above)
Comment (azexcy) [reply kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java:73]: Add runtime exception cache,  It's not thrown exception usually.

**Review by sandynz — APPROVED** [review]

File: kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java [hunk 3]
```diff
@@ -64,22 +64,22 @@ public void onInitialized(final String databaseName, final ContextManager contex
...
+        } catch (final IllegalArgumentException | UnsupportedOperationException | PipelineJobNotFoundException ex) {
             log.error("Dispatch enable pipeline job start event failed", ex);
         }
     }
     
     private void dispatchEnablePipelineJobStartEvent(final PipelineContextKey contextKey) {
         JobConfigurationAPI jobConfigAPI = PipelineAPIFactory.getJobConfigurationAPI(contextKey);
         List<JobBriefInfo> allJobsBriefInfo = PipelineAPIFactory.getJobStatisticsAPI(contextKey).getAllJobsBriefInfo()
-                .stream().filter(each -> !each.getJobName().startsWith("_")).collect(Collectors.toList());
+                .stream().filter(each -> !each.getJobName().startsWith("_") && !each.getJobName().startsWith("j02")).collect(Collectors.toList());
```
Comment (sandynz) [inline kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java:75]: It's better not hard-code job name prefix

**Review by sandynz — CHANGES_REQUESTED** [review]

**Review by sandynz — APPROVED** [review]

**Pull request merged**